### DISCUSSION TITLE
DAI 2023-2024 - Practical work 3 - Lucas Lattion and Romain Humair

### DISCUSSION THREAD

**Discussion by HumairRomain** [heig-vd-dai-course/heig-vd-dai-course#162] [Show and tell]
Système de tracking.

Des trackers GPS (multicast) envoient leurs données à un système de tracking :
* timestamp
* latitude / longitude
* niveau de batterie

Des clients (unicast) peuvent récupérer les données de certains trackers : 
* tracé (historique de déplacement) (fichier csv ?)
* dernière position
* position à un temps donné
* vitesse min / max / moyenne
* distance
* niveau de batterie du tracker

Ajoute possible (broadcast)
* Alerte geofence si un tracker sort d'une zone délimitée

**Comment by HumairRomain** [comment]
repo : https://github.com/lucaslattion/DAI-2023-24-Practical-work-3-Lucas-Lattion-and-Romain-Humair

**Comment by lucaslattion** [comment]
https://github.com/lucaslattion/DAI-2023-24-Practical-work-3-Lucas-Lattion-and-Romain-Humair/tree/a264ae6a4e80ffa90ed652ac193b764b1139edc9

Hash: a264ae6a4e80ffa90ed652ac193b764b1139edc9
